apps/charterafrica/src/payload/fields/updatedAt.js

```diff
@@ -0,0 +1,22 @@
+import { deepmerge } from "@mui/utils";
```

Comment:
Isn't this very similar to deletedAt with exception of `beforeValidate` hook? Can you extend that one instead of completely rewriting it?